extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -53,7 +53,8 @@ export class SchemaCompareResult {
 		this.SchemaCompareActionMap[azdata.SchemaUpdateAction.Change] = localize('schemaCompare.changeAction', 'Change');
 		this.SchemaCompareActionMap[azdata.SchemaUpdateAction.Add] = localize('schemaCompare.addAction', 'Add');
 
-		this.editor = azdata.workspace.createModelViewEditor(localize('schemaCompare.Title', 'Schema Compare'), { retainContextWhenHidden: true, supportsSave: true });
+		// Do not localize paramater customResourceName
+		this.editor = azdata.workspace.createModelViewEditor(localize('schemaCompare.Title', 'Schema Compare'), { retainContextWhenHidden: true, supportsSave: true, customResourceName: 'Schema Compare' });
```

Comment:
another thought, since you were able to set the title of the schema compare editor window's title to Schema Compare without needing this before, I guess the title and the resource name is not tied to each other, i am thinking you can give a more id/name like value here, e.g. _schema_compare. by doing this: 1. people won't think this value is localizable 2. it is unlikely that people will have a file named like this.  thoughts?

Reply:
good point.. will try with _schema_compare

Reply:
as discussed leaving this one as is & documenting with comments